linux-static/2_build_toolchain.sh

```diff
@@ -0,0 +1,161 @@
+#!/bin/bash
+
+# abort on error
+set -e
+
+export WORKSPACE=$PWD
+
+# Number of CPU
+nproc=$(nproc)
+
+# Use ccache?
+if [ -z ${NO_CCACHE+x} ]; then
+	if hash ccache >/dev/null 2>&1; then
+		ENABLE_CCACHE=1
+		echo "CCACHE enabled"
+	fi
+fi
+
+if [ ! -f .patches-applied ]; then
+	echo "Patching libraries"
+
+	# disable pixman examples and tests
+	cd pixman-0.34.0
+	perl -pi -e 's/SUBDIRS = .*/SUBDIRS = pixman/' Makefile.am
+	autoreconf -fi
+	cd ..
+
+	# disable png utils
+	cd libpng-1.6.34
+	perl -pi -e 's/^bin_PROGRAMS.*//' Makefile.am
+	autoreconf -fi
+	cd ..
+
+	# disable harfbuzz utils and tests
+	cd harfbuzz-1.5.1
+	perl -pi -e 's/SUBDIRS = .*/SUBDIRS = src/' Makefile.am
+	autoreconf -fi
+	cd ..
+
+	# disable unsupported compiler flags by clang in libvorbis
+	perl -pi -e 's/-mno-ieee-fp//' libvorbis-1.3.5/configure
+
+	# fix building
+	patch -Np1 -d libxmp-lite-4.4.1 < libxmp-a0288352.patch
+
+	# disable libsndfile examples and tests
+	cd libsndfile-1.0.28
+	perl -pi -e 's/ examples regtest tests programs//' Makefile.am
+	autoreconf -fi
+	cd ..
+
+	# Wildmidi: Disable libm
+	perl -pi -e 's/FIND_LIBRARY\(M_LIBRARY m REQUIRED\)//' wildmidi-wildmidi-0.4.1/CMakeLists.txt
```

Comment:
Will try to upstream a version that does not use REQUIRED because this fails everywhere...